### PR TITLE
POST /spender/expenses/check_mandatory_fields

### PR DESCRIPTION
**Pull request by KirtiGautam**
image

image

image

image

### CODE REVIEW

File: src/components/schemas/expense.yaml [hunk 1]
```diff
@@ -1114,4 +1114,127 @@ expenses_stats_out:
...
       description: |
         This field indicates the sum of amount of expenses that match the criteria specified in the query_params within the body of the request.
-      example: 100
\ No newline at end of file
+      example: 100
+
+spender_expense_check_mandatory_fields_in:
+  type: object
+  additionalProperties: false
+  properties:
```
Comment (kartikeyrajvaidya) [inline src/components/schemas/expense.yaml:1122]: why not use expense_in here?

Comment (KirtiGautam) [reply src/components/schemas/expense.yaml:1122]: There is an additional field `num_files` which is not present in the expenses_in. Also, validations here are different.

Comment (KirtiGautam) [reply src/components/schemas/expense.yaml:1122]: Reference expenses_in and make changes

**Review by kartikeyrajvaidya — APPROVED** [review]

File: src/components/schemas/expense.yaml (hunk 1 above)
Comment (KirtiGautam) [reply src/components/schemas/expense.yaml:1122]: Referencing `expenses_in` is not allowing us to change the required fields. Going with creating a new schema. [Ref](https://fylein.slack.com/archives/C055191R3AS/p1691665050382289?thread_ts=1691466533.685379&cid=C055191R3AS)

**Pull request merged**